tests/suites/cli/active_branch.sh

```diff
@@ -0,0 +1,88 @@
+# Checks juju indicates the correct branch output tabular and json.
+run_indicate_active_branch_no_active() {
+  echo
+
+  file="${TEST_DIR}/indicate-active-branch-no-active.txt"
+
+  ensure "indicate-active-branch-no-active" "${file}"
+
+  export JUJU_DEV_FEATURE_FLAGS=generations
+  juju deploy ubuntu 2>&1
+
+  wait_for "ubuntu" ".applications | keys[0]"
+  STATUS_NO_BRANCHES=$(juju status)
+  STATUS_JQ_NO_BRANCHES=$(juju status --format=json | jq '.branches')
+  check_not_contains "${STATUS_NO_BRANCHES}" "Branch"
+
+  if [ "${STATUS_JQ_NO_BRANCHES}" != null ]; then
```

Comment:
Considering we don't reuse the variable can we just in line the call?

```suggestion
  if [ "$(juju status --format=json | jq '.branches')" != null ]; then
```